.github/workflows/test_build.yml

```diff
@@ -0,0 +1,88 @@
+name: Release
+
+on: [ push, pull_request ]
+
+jobs:
+  # the create-release would be used once we want to create a specific release based on if the commit has a tag
+  create-release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        env:
+          # (required)
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  
+  # build-happ:
+  #   runs-on: ubuntu-latest
+  #   steps:
+  #     - uses: actions/checkout@v2
+  #     # install hc tool
+  #     - name: Install hc tool and wasm target
+  #       shell: bash # important because this runs shell scripts
+  #       run: |
+  #         npm run happ-install
+  #     # build hApp
+  #     - name: Build hApp
+  #       run: |
+  #         npm run happ-pack
+  #     # "upload" dna as build artifact
+  #     - uses: actions/upload-artifact@master
+  #       with:
+  #         name: profiles-artifact
+  #         path: happ/workdir/profiles.happ
+  #     - uses: actions/upload-artifact@master
+  #       with:
+  #         name: projects-artifact
+  #         path: happ/workdir/projects.dna
+
+  # upload the app package for each target os
+  upload-assets:
+    # needs: build-happ
+    strategy:
+      matrix:
+        os:
+          - ubuntu-18.04
```

Comment:
this one can be updated to `ubuntu-latest`. We use to have that old one to try to maintain more backwards compatibility

Reply:
in the `test_build.yml` or `release.yml`? @Connoropolous

Reply:
both